recipe/meta.yaml

```diff
@@ -1,42 +1,48 @@
 {% set name = "pytest-metadata" %}
-{% set version = "3.0.0" %}
+{% set version = "2.0.4" %}
```

Comment:
Why not go with the latest? The latest is 3.1.1 and `pytest-json-report` doesn't seem to have any particular requriements regarding `pytest-metadata`.

Reply:
pytest-json-report has a bug with the latest version that is discovered when running the pytests

https://github.com/numirias/pytest-json-report/issues/89#issuecomment-1760092492

Once pytest-json-report launches a stable version with the changes they made for the current version, we will move away from 2.0.4